### DISCUSSION TITLE
Config nginx with Hide my wp Ghost pluging

### DISCUSSION THREAD

**Discussion by jcuc** [EasyEngine/easyengine#1612] [Support Requests]
Can anyone help me to configure Hidemywpghost plugin with easyengine 4? 

I have to add this line
/opt/easyengine/sites/example.com/app/htdocs/hidemywpghost.conf;

to nginx configuration example.com

So I have tried adding it to /opt/easyengine/sites/example.com/config/nginx/nginx.conf
and the web does not work, error 502

So I have tried on
/opt/easyengine/sites/example.com/config/nginx/conf.d/main.conf
and the web does not work, error 502

And then I have tried on
/opt/easyengine/sites/example.com/config/nginx/custom/user.conf

and the web does not work, error 502

Can anybody help me?

**Comment by dhsathiya** [comment]
Can you put the configuration here that you want to add?

**Reply by jcuc** [reply]
I have to put the line 
Include /opt/easyengine/sites/example.com/app/htdocs/hidemywpghost.conf

**Reply by nachocheeseburger** [reply]
Have you figured this out? I have the same issue

**Comment by jcuc** [comment]
I finally gave up, I didn't get support.
________________________________
De: nachocheeseburger ***@***.***>
Enviado: jueves, 20 de octubre de 2022 23:37
Para: EasyEngine/easyengine ***@***.***>
Cc: jcuc ***@***.***>; Author ***@***.***>
Asunto: Re: [EasyEngine/easyengine] Config nginx with Hide my wp Ghost pluging (Discussion #1612)


Have you figured this out? I have the same issue

**Reply by nachocheeseburger** [reply]
I figured it out :)

edit this file "/opt/easyengine/sites/example.com/config/nginx/conf.d/main.conf"

add line "include /var/www/htdocs/hidemywp.conf;"
